### DISCUSSION TITLE
Logo in the main page

### DISCUSSION THREAD

**Discussion by riramar** [OWASP/www-project-secure-headers#208] [Ideas]
@righettod I loved the new AI logos. How about put one of them in the main project page?
For me the perfect would be a combination of the logo04-microsoft-copilot.png image with the writing of logo01-chatgpt.png.
What do you think?

https://owasp.org/www-project-secure-headers/logo/generated-via-ai/logo04-microsoft-copilot.png

https://owasp.org/www-project-secure-headers/logo/generated-via-ai/logo01-chatgpt.png

**Comment by righettod** [comment]
I will see what I can do using your directives and let you know 😉

**Comment by righettod** [comment]
💻 On Copilot, I cannot upload image to use the source image that specified:

https://support.microsoft.com/en-us/topic/file-formats-supported-by-copilot-1afb9a70-2232-4753-85c2-602c422af3a8 

![image](https://github.com/user-attachments/assets/3738908b-0a83-4c69-918a-fa28b7d34dbe)

💻 On ChatGPT, I quickly reach the daily limitation, I used the following prompt:

```text
Add the text "OWASP Secure Headers Project" from the image attached in file "source.png" to the image attached in file "base.png" using the same design for the text.
```

**BASE:**

![base](https://github.com/user-attachments/assets/0aece5cf-e409-47b5-a08a-aaae211969c2)

**SOURCE:**

![source](https://github.com/user-attachments/assets/ed047d3e-9801-4c7b-ad93-3e6b6953f1a2)

👨‍💻 I will continue when my locking period is finished.

**Comment by righettod** [comment]
@riramar 
I generated a new one, with ChatGPT and Microsoft Copilot with only the project name, and using this prompt:

```text
Generate a image that will be a logo.
The image is a fortress from a cybersecurity perspective.
The sentence "OWASP Secure Headers Project" is around the image.
```

**ChatGPT**

![tentative01-chatgpt](https://github.com/user-attachments/assets/a180593a-f7cb-48fd-a5e7-8320d54c6be8)

**Microsoft Copilot**

![tentative02-microsoft-copilot](https://github.com/user-attachments/assets/d3a1fc2d-1fea-4cdc-a000-5dcefc8e84e7)

![tentative03-microsoft-copilot](https://github.com/user-attachments/assets/a808ddb4-62a8-4aa1-b27e-e54b47186692)

[image-files.zip](https://github.com/user-attachments/files/18222484/image-files.zip)

🤔ChatGPT seems more effective in such task than Microsoft Copilot but perhaps my prompt is not well defined/structured.

😊Personally I like the new one generated by ChatGPT and we could use it on the main page if it suit you too.

**Comment by riramar** [comment]
I agree with you. I liked the one below.
If you want we can insert it in our main page.

![image](https://github.com/user-attachments/assets/3f2c70a4-1877-4ea9-b935-9383d874187f)

**Comment by righettod** [comment]
PR #209 submitted to add the new generated logos as well as the logo on the main page.

PR https://github.com/OWASP/owasp-swag/pull/72 submitted to add new generated logos.

**Comment by righettod** [comment]
I also updated the OSHP org metadata:

![image](https://github.com/user-attachments/assets/842cb885-0bdc-48dc-8cd2-2319fdea3715)

**Comment by riramar** [comment]
Thanks!
